base/prompts/ralph-loop-v2.md

```
@@ -0,0 +1,124 @@
+### Step 6: Stuck detection
+If you've attempted fixes 3+ times and CI still fails, or you can't resolve a review comment:
+- Print: `BLOCKED: {{description of what's blocking}}`
+- Print: `ATTEMPTED: {{list of what you tried}}`
+- Do NOT keep looping — signal and stop
+
+## Git Config
+```bash
+git config user.name "kaylee-mistystep"
+git config user.email "kaylee@mistystep.io"
+```
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)